.github/workflows/build.yml

```diff
@@ -24,21 +24,16 @@ jobs:
         mkdir -p ~/.docker
         echo "{}" > ~/.docker/config.json
         touch ${HOME}/.gitconfig
-    - name: build
-      run: |
-        skipper make
-    - name: Get release version
-      id: get_version
-      run: echo ::set-env name=GIT_REVISION::$(echo ${GITHUB_SHA})
-    - name: Publish to Registry
+    - name: Publish build to Registry
```

Comment:
I don't understand this change.
What are you trying to do here?

Reply:
This was just for a test for the issues that we had on Wednesday